src/test/java/org/openmrs/contrib/qaframework/automation/ReportSteps.java

```diff
@@ -0,0 +1,92 @@
+package org.openmrs.contrib.qaframework.automation;
+
+import org.openmrs.contrib.qaframework.RunTest;
+import org.openmrs.contrib.qaframework.page.AdministrationPage;
+import org.openmrs.contrib.qaframework.page.ManageReportsPage;
+import org.openmrs.contrib.qaframework.page.RenderDefaultReportPage;
+import org.openmrs.contrib.qaframework.page.ReportHistoryPage;
+import org.openmrs.contrib.qaframework.page.RunReportPage;
+import org.openmrs.contrib.qaframework.page.SystemAdministrationPage;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class ReportSteps extends Steps {
+
+	private SystemAdministrationPage systemAdministrationPage;
+	private AdministrationPage administrationPage;
+	private ManageReportsPage manageReportsPage;
+	private ReportHistoryPage reportHistoryPage;
+	private RunReportPage runReportsPage;
+	private RenderDefaultReportPage renderDefaultReportPage;
+	private static final String startDate = "05 / 07 / 2008";
+	private static final String endDate = "20 / 06 / 2020";
```

Comment:
did you just forget to align the static final variable names to right case letter?